CHANGELOG.rst

```diff
@@ -4,6 +4,23 @@ Community ProxySQL Collection Release Notes
 
 .. contents:: Topics
 
+
+v1.6.1
+======
+
+Release Summary
+---------------
+
+This is the minor release of the ``community.proxysql`` collection.
+This changelog contains all changes to the modules and plugins in this collection
+that have been made after the previous release.
+
+Minor Changes
+-------------
+
+- proxysql_mysql_users role - Creating users with the caching_sha2_password plugin (https://github.com/ansible-collections/community.proxysql/pull/164).
+
+
```

Comment:
```suggestion

```

This is autogenerated later in the release process using `antsibull-changelog`.  
All that is necessary it to place a changelog file to `changelogs/fragments`.  

You can cheat on more active collections how it looks like: https://github.com/ansible-collections/community.general/tree/main/changelogs/fragments